### PR TITLE
NEW Add possibility to choose format #21426

### PR DESCRIPTION
**Pull request by KiVoilaJe**
NEW|New feature(#21426) Added in the export module the possibility to choose between utf-8 and iso format

### CODE REVIEW

File: htdocs/core/modules/export/export_csvutf8.modules.php [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+
+require_once DOL_DOCUMENT_ROOT.'/core/modules/export/modules_export.php';
+
+// avoid timeout for big export
+set_time_limit(0);
+
+/**
+ *	Class to build export files with format CSV
+ */
+class ExportCsvUtf8 extends ModeleExports
```
Comment (eldy) [inline htdocs/core/modules/export/export_csvutf8.modules.php:32]: It seems this class is a copy of the class for iso but with just few lines changed.
So why not don't you make an inherit 
 class ExportCsvUtf8 extends ExportCsv
And implement into class only properties or method with change for utf8 ?

**Comment by Hystepik** [comment]
Still travis error. Try to move the set of the CHARSET... into an surcharging method write_record, ... instead of into constructor.

**Comment by Hystepik** [comment]
Unit test to fix

**Comment by Hystepik** [comment]
@eldy seams ok

**Pull request merged**